modules/admin_manual/pages/configuration/user/oidc/index.adoc

```diff
@@ -3,56 +3,188 @@
 :toclevel: 2
 :openid-connect-frontchannel-logout-url: https://openid.net/specs/openid-connect-frontchannel-1_0.html
 :openid-connect-url: https://openid.net/connect/
+:openid-connect-app: https://marketplace.owncloud.com/apps/openidconnect
+:konnect: https://github.com/Kopano-dev/konnect
+:konnect-docs: https://github.com/Kopano-dev/konnect#running-konnect
+:konnect-webserver: https://documentation.kopano.io/kopanocore_administrator_manual/configure_kc_components.html#configure-a-webserver-for-konnect
 
 == Introduction
 
-"OpenID Connect 1.0 is a simple identity layer on top of the OAuth 2.0 protocol. It allows Clients to verify the identity of the End-User based on the authentication performed by an Authorization Server, as well as to obtain basic profile information about the End-User in an interoperable and REST-like manner.
-OpenID Connect allows clients of all types, including Web-based, mobile, and JavaScript clients, to request and receive information about authenticated sessions and end-users. The specification suite is extensible, allowing participants to use optional features such as encryption of identity data, discovery of OpenID Providers, and session management, when it makes sense for them."
--- {openid-connect-url}, "What is OpenID Connect?"
+{openid-connect-url}[OpenID Connect] is an open standard for single sign-on and identity and access management. With ownCloud it can be used for user authentication and client authorization against an external Identity Provider.
+
+**Benefits of using ownCloud with OpenID Connect**
+
+- Increased security by shifting user authentication to an external Identity Provider
+- Seamless integration into single sign-on (SSO) environments as well as with third party products
+- Centralized client management within the Identity Provider
+- Enterprise-grade security through the use of authentication security features (e.g., multi-factor authentication) and policies (e.g., automatic token expiration on certain conditions) provided by Identity Providers
 
 == Prerequisites
+- OpenID Connect requires an external Identity Provider
+- A distributed memcache setup is required to operate this app - such as Redis or Memcached. Please follow ref:configuration/server/caching_configuration.adoc[the documentation] on how to set up caching.
```

Comment:
```suggestion
- A distributed memcache setup is required to operate this app - such as Redis or Memcached. Please follow xref:configuration/server/caching_configuration.adoc[the documentation] on how to set up caching.
```

Reply:
Fixed in PR #2837 and backport fixed in #2838